### ISSUE TITLE
2.6.564: YamlReader#readList:28 - java.lang.ClassCastException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#10091]
## User Description
just got a brand new mac which is running macOS Monterey and when i tried running latest tripleA client (pressing Play after loading a downloaded game and filling in my creds for aaa.org), it doesn't seem to log me in and load the game. the Play butten then grays out and i can't do anything. i also get an error when i try to do a Test Post (this error i'm reporting).

## Map
world_war_ii_global

## Log Message
Invalid yaml format: class java.util.LinkedHashMap cannot be cast to class java.util.List (java.util.LinkedHashMap and java.util.List are in module java.base of loader 'bootstrap')

## TripleA Version
2.6.564

## Java Version
11.0.9.1

## Operating System
Mac OS X

## Stack Trace
```
Exception: org.triplea.yaml.YamlReader$InvalidYamlFormatExceptionInvalid yaml format: class java.util.LinkedHashMap cannot be cast to class java.util.List (java.util.LinkedHashMap and java.util.List are in module java.base of loader 'bootstrap')
java.lang.Exception
	at org.triplea.yaml.YamlReader.readList(YamlReader.java:28)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.uploadSaveGame(NodeBbForumPoster.java:159)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.post(NodeBbForumPoster.java:125)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.postTurnSummary(NodeBbForumPoster.java:103)
	at games.strategy.engine.framework.startup.ui.posted.game.pbf.test.post.TestPostAction.lambda$accept$1(TestPostAction.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)


Exception: java.lang.ClassCastExceptionclass java.util.LinkedHashMap cannot be cast to class java.util.List (java.util.LinkedHashMap and java.util.List are in module java.base of loader 'bootstrap')
java.lang.Exception
	at org.triplea.yaml.YamlReader.readList(YamlReader.java:26)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.uploadSaveGame(NodeBbForumPoster.java:159)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.post(NodeBbForumPoster.java:125)
	at games.strategy.engine.posted.game.pbf.NodeBbForumPoster.postTurnSummary(NodeBbForumPoster.java:103)
	at games.strategy.engine.framework.startup.ui.posted.game.pbf.test.post.TestPostAction.lambda$accept$1(TestPostAction.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)


```

**Comment by beelee1** [comment]
Bummer probably best to go back to 2.5 

Been some new node stuff going on recently that doesn't work with triplea. Macs always been a bit problematic

**Comment by DanVanAtta** [comment]
This is fixed in the latest 2.6
